PF/JUnitTest/AreaTesting.java

```diff
@@ -0,0 +1,15 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class AreaTesting {
+
+    @Test
+    public void testArea() {
```

Comment:
test method naming convention should be like test+ method name + method desciption